### PR TITLE
[Edit] Kotlin: removePrefix - fix entry style

### LINKED ISSUE #3820: [Edit] Kotlin: removePrefix - fix entry style
### Type of Edit (select all that apply)

Fix grammar, spelling, formatting

### Description (optional)

The Kotlin [removePrefix](https://www.codecademy.com/resources/docs/kotlin/strings/removePrefix) method term entry needs to be updated to match the overall style of the other term entries for Kotlin Strings.

I propose to do the following:
- Add the missing dot to the mentions of the removePrefix method in the description and example (3 occurrences in total, ```removePrefix``` should be ```.removePrefix```).
- Simplify syntax code snippet from ```val modifiedString = originalString.removePrefix(prefixToRemove)``` to ```String.removeSuffix(suffix)```.
- Update variables names in the syntax section to match the new snippet.
- Update example code snippet to use the same originalString variable value ("HelloWorld.jpg") as the [removeSuffix](https://www.codecademy.com/resources/docs/kotlin/strings/removeSuffix) method term entry page. The example code snippet will also be updated to reflect the change to the originalString variable.
- (Optional) Remove empty line between suffix and modifiedString variables in the example code snippet and add an empty line after the modifiedString variable to honor [Kotlin style guide](https://developer.android.com/kotlin/style-guide#vertical).

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by IvanT98**
### Description

The Kotlin [removePrefix](https://www.codecademy.com/resources/docs/kotlin/strings/removePrefix) method term entry needed to be updated to match the overall style of the other term entries for Kotlin Strings.

The following was done:
- Added the missing dot to the mentions of the removePrefix method in the description and example (```removePrefix``` should be ```.removePrefix```).
- Simplified syntax code snippet from ```val modifiedString = originalString.removePrefix(prefixToRemove)``` to ```String.removeSuffix(suffix)```.
- Updated variables names in the syntax section to match the new snippet.
- Updated example code snippet to use the same originalString variable value ("HelloWorld.jpg") as the [removeSuffix](https://www.codecademy.com/resources/docs/kotlin/strings/removeSuffix) method term entry page. The example code snippet was also be updated to reflect the change to the originalString variable.
- Removed empty line between suffix and modifiedString variables in the example code snippet and add an empty line after the modifiedString variable to honor [Kotlin style guide](https://developer.android.com/kotlin/style-guide#vertical).
- Updated entry description to add 'Kotlin' before 'string'.

Closes #3820.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/kotlin/concepts/strings/terms/removePrefix/removePrefix.md [hunk 1]
```diff
@@ -13,33 +13,33 @@ CatalogContent:
   - 'paths/computer-science'
 ---
 
-The **`removePrefix()`** method in Kotlin is used to remove a specified prefix from a string if it is present. The `removePrefix()` method returns a new string with the specified prefix removed if it exists. If the original string does not start with the specified prefix, the method returns the original string unchanged.
+The **`.removePrefix()`** method in Kotlin is used to remove a specified prefix from a string if it is present. The `.removePrefix()` method returns a new string with the specified prefix removed if it exists. If the original string does not start with the specified prefix, the method returns the original string unchanged.
```
Comment (avdhoottt) [inline content/kotlin/concepts/strings/terms/removePrefix/removePrefix.md:16]:
```suggestion
The **`.removePrefix()`** method in Kotlin is used to remove a specified prefix from a string if it is present. It returns a new string with the specified prefix removed if it exists. If the original string does not start with the specified prefix, the method returns the original string unchanged.
```
The function name seems repetitive.

**Review by avdhoottt — CHANGES_REQUESTED** [review]

**Review by avdhoottt — APPROVED** [review]
PR is Ready for the Second Review.

File: content/kotlin/concepts/strings/terms/removePrefix/removePrefix.md (hunk 1 above)
Comment (IvanT98) [reply content/kotlin/concepts/strings/terms/removePrefix/removePrefix.md]: Good suggestion. Added

**Comment by IvanT98** [comment]
Pushed new commit to remove space from new line in the example code snippet that was causing the format:verify step to fail

**Review by SSwiniarski — APPROVED** [review]
@IvanT98 Looks good.

**Pull request merged**